src/components/EventCard.js

```diff
@@ -40,20 +80,51 @@ const Title = styled.h3`
   margin-top: 0;
 `
 
-const EventCard = ({ title, to, date, description, className, location }) => (
-  <StyledCard className={className}>
-    <Date>
-      {date}
-      <Emoji text=":spiral_calendar:" size={1} ml={`0.5em`} />
-    </Date>
-    <Location>
-      <LocationText>{location}</LocationText>
-      <Emoji text=":round_pushpin:" size={1} ml={`0.5em`} />
-    </Location>
-    <Title>{title}</Title>
-    <Description>{description}</Description>
-    <ButtonLink to={to}>View Event</ButtonLink>
-  </StyledCard>
-)
+const EventCard = ({
+  title,
+  to,
+  date,
+  description,
+  className,
+  location,
+  idx,
+}) => {
+  const isEven = (idx + 1) % 2 === 0
+  return (
+    <StyledCard className={className}>
+      <StyledCardReference />
+      {!isEven && (
+        <StyledCardContent>
+          <Date>
+            {date}
+            <Emoji text=":spiral_calendar:" size={1} ml={`0.5em`} />
+          </Date>
+          <Location>
+            <LocationText>{location}</LocationText>
+            <Emoji text=":round_pushpin:" size={1} ml={`0.5em`} />
+          </Location>
+          <Title>{title}</Title>
+          <Description>{description}</Description>
+          <ButtonLink to={to}>View Event</ButtonLink>
+        </StyledCardContent>
+      )}
+      {isEven && (
+        <StyledCardContentRight>
+          <Date>
+            {date}
+            <Emoji text=":spiral_calendar:" size={1} ml={`0.5em`} />
+          </Date>
+          <Location>
+            <LocationText>{location}</LocationText>
+            <Emoji text=":round_pushpin:" size={1} ml={`0.5em`} />
+          </Location>
+          <Title>{title}</Title>
+          <Description>{description}</Description>
+          <ButtonLink to={to}>View Event</ButtonLink>
+        </StyledCardContentRight>
+      )}
+    </StyledCard>
```

Comment:
Couple thoughts here on ways to reduce redundant code...
1- Pass `isEven` as a prop to `StyledCardContent` (`<StyledCardContent isEven={isEven}>`) and then pick this up in the CSS to conditionally render on the left or right.
2- Could try using `&:nth-child(even)` from the parent to style all even children differently. 
3- Could otherwise declare a constant before the return in this component that holds all the children that are currently written twice, then reuse that constant (as a mini component) in the final return. 

The CSS approach I think would be the cleanest, but otherwise the first suggestion would suffice and would remove the need to have this duplicated.

Reply:
Nice! Thanks for the suggestions, will work on them 🙂